app/Http/Controllers/AnswerVoteController.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\AnsVotes;
+use Illuminate\Http\Request;
+
+class AnswerVoteController extends Controller
+{
+    public function store(Request $request, $question_id, $answer_id, $vote)
```

Comment:
merge these route parameter with route parameters and add validation for them as well